bitacross-worker/enclave-runtime/src/lib.rs

```diff
@@ -486,203 +476,6 @@ pub unsafe extern "C" fn migrate_shard(
 	sgx_status_t::SGX_SUCCESS
 }
 
-#[no_mangle]
-pub unsafe extern "C" fn sync_parentchain(
-	blocks_to_sync: *const u8,
-	blocks_to_sync_size: usize,
-	events_to_sync: *const u8,
-	events_to_sync_size: usize,
-	events_proofs_to_sync: *const u8,
-	events_proofs_to_sync_size: usize,
-	parentchain_id: *const u8,
-	parentchain_id_size: u32,
-	immediate_import: c_int,
-) -> sgx_status_t {
-	if let Err(e) = sync_parentchain_internal(
-		blocks_to_sync,
-		blocks_to_sync_size,
-		events_to_sync,
-		events_to_sync_size,
-		events_proofs_to_sync,
-		events_proofs_to_sync_size,
-		parentchain_id,
-		parentchain_id_size,
-		immediate_import == 1,
-	) {
-		error!("Error synching parentchain: {:?}", e);
-		return sgx_status_t::SGX_ERROR_UNEXPECTED
-	}
-
-	sgx_status_t::SGX_SUCCESS
-}
-
-#[allow(clippy::too_many_arguments)]
-unsafe fn sync_parentchain_internal(
```

Comment:
it seems that with removal of this code omits the event `teebag.ParentchainBlockProcessed` which is triggered every parachain block. I've removed this logic since it was referenced to `bitacross-worker/core-primitives/enclave-api/src/sidechain.rs` and I'm not sure  do we want to initiate this event or not? 
@kziemianek @felixfaisal , do you have any suggestions?

Reply:
IMO we'll need this event - it signals the processing of parentchain block syncing